user_paths_example.py

```diff
@@ -1,4 +0,0 @@
-# Copy and paste your address bar contents for each of your desired locations here. 
-# Remove "_example" from the name of this file.
-MISSIONS_PATH = r"E:\Documents\Arma 3\MPMissions"
-PARADIGM_PATH = r"E:\Documents\Arma 3\SGD_Paradigm"
```

Comment:
Reintroduce this if possible please, just makes merging smoother

Reply:
Aight. I have now done so. Also no worries about the delay.